tardis/grid/__init__.py

```diff
@@ -0,0 +1 @@
+from tardis.grid.tardisgrid import *
```

Comment:
* imports are a bit risky, are you sure this is okay?

Reply:
This is just so that the class can be imported using the statement `import tardis.grid`. I think most of the directories in tardis with __init__.py files do something similar.

Reply:
But I can just as easily list everything out.  That's definitely safer in case the grid code grows.

Reply:
Going to leave as `*` import since this is the precedent in all other major modules.  We should discuss changing this practice in its own PR.